.github/workflows/gradle-build-deploy.yml

```diff
@@ -55,4 +53,4 @@ jobs:
 
     - name: Copy Files to the Website
       run: |
-        aws s3 sync ./site/* s3://modsurski.com/csci162 --delete
+        aws s3 sync . s3://modsurski.com/csci162 --delete
```

Comment:
I don't really trust uploading `.`, we don't really know if github will try to be helpful and put some crap into whatever directory we're in here. Downloading the artifacts to a new directory and uploading that is likely to be safer, imo.